### ISSUE TITLE
Testing Custom Runtime Requirements

### ISSUE THREAD

**Issue by sitapriyamoorthi** [FredHutch/wdl-unit-tests#15]
Custom runtime resources: Tests the ability to specify custom resources (CPU, memory) in the runtime block.
Docker usage with runtime: Ensures Docker containers are correctly handled when custom resources are requested.

**Comment by tefirman** [comment]
@sitapriyamoorthi -- I believe this functionality is demonstrated in "helloDockerHostname.wdl", do you agree?

**Comment by sitapriyamoorthi** [comment]
@tefirman yes i think the "helloDockerHostname.wdl" does test the ability to spin up a docker and use of the specified memory allocations. 

I guess what is missing in this test (or could be added into the WDL) would be testing these use cases: 

1. Testing dynamic run time variables ( a user provides dynamic values as inputs) and making sure that these are actually allocated?
2. Checking if custom runtime variables are actually used ?
3.  Invalid runtime variables returning a valid meaningful error (but a successful WDL run)

Totally might be an overkill but would be a potentially good way to monitor resource allocations are appropriately being given on the cluster. 

I know we also need to test the GPU allocations but I know that there is an issue with testing those with GitHub actions and we potentially needs to create a test and independently check this?
